web_teleop/nodes/actuators_node.py

```diff
@@ -21,45 +18,50 @@ def __init__(self):
         self._head = robot_api.Head()
         self._gripper = robot_api.Gripper()
         self._arm = robot_api.Arm()
+
     def handle_set_torso(self, request):
+        # type: (SetTorsoRequest) -> None
```

Comment:
Remove commented out code (this is also a really small thing so I can live with this)

Reply:
this is a type hint